sqllogictest-engine/Cargo.toml

```diff
@@ -0,0 +1,33 @@
+[package]
+name = "sqllogictest-engine"
+version = { workspace = true }
```

Comment:
We can begin to use separated versions? 🤪

Reply:
I'm not sure, but it seems not bad for using a unified version 🤣 which is less ambiguous for users.